packages/appsync-emulator-serverless/__test__/example/serverless.yml

```diff
@@ -251,6 +275,11 @@ resources:
                     - 'lambda:invokeFunction'
                   Resource:
                     - { Fn::GetAtt: [GraphqlLambdaFunction, Arn] }
+                - Effect: 'Allow'
+                  Action:
+                    - 'lambda:invokeFunction'
+                  Resource:
+                    - { Fn::GetAtt: [PythonLambdaFunction, Arn] }
```

Comment:
I think? we want 

```
                    - { Fn::GetAtt: [GraphqlLambdaFunction, Arn] }
```

Reply:
Think get this recommendation :)

Reply:
Not sure what you mean

Reply:
I think I cannot get this recommendation, because i didn't understand it :)

Reply:
I think instead of 
```- { Fn::GetAtt: [PythonLambdaFunction, Arn] }```
we want
```- { Fn::GetAtt: [GraphqlLambdaFunction, Arn] }```

Reply:
@danifbento I am not an expert on this section of the Resources either.  I think my suggestion is the correct one.  Update it and I am happy to merge.

It would also be helpful if you bumped the version for this package 
 to `0.7.11` : https://github.com/ConduitVC/aws-utils/blob/eaaa420ee4085699cde5ad8c60abbee68ab9643a/packages/appsync-emulator-serverless/package.json#L3

Reply:
The `- { Fn::GetAtt: [GraphqlLambdaFunction, Arn] }` is already there, I was trying to give policy to the `PythonLambdaFunction` too or it isnt necessary?

Reply:
Ahh, I don't think `PythonLambdaFunction` is a thing, I think it will be covered by ` GraphqlLambdaFunction`

Reply:
Done, I removed and maintain the one that is already there :)